core/src/blockstream.rs

```diff
@@ -91,7 +93,17 @@ where
         leader_id: &Pubkey,
         entry: &Entry,
     ) -> Result<()> {
-        let json_entry = serde_json::to_string(&entry)?;
+        let transactions: Vec<Vec<u8>> = entry
+            .transactions
+            .iter()
+            .map(|tx| serialize(&tx).unwrap())
```

Comment:
This nicely de-risks https://github.com/solana-labs/solana/pull/3545, but could I replace that `serialize()` call with an empty string and still see the test suite pass?

Reply:
No, `test_blockstream_service_process_entries()` will definitely fail trying to deserialize that empty Vec into the original transaction. However, I can make the testing more explicit.